app/models/petition.rb

```diff
@@ -100,6 +102,10 @@ class Petition < ActiveRecord::Base
   scope :with_response, -> { where.not(response_summary: nil, response: nil) }
   scope :with_debate_outcome, -> { joins(:debate_outcome) }
 
+  scope :without_debate_date, -> { where.not(id: DebateOutcome.select(:petition_id).uniq) }
```

Comment:
Maybe change this to ``without_debate_outcome`` so it's clearer that it's the inverse of that scope?